plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/app/PluginStrings.js

```diff
@@ -740,7 +740,7 @@ Ext.define('NX.coreui.app.PluginStrings', {
     System_HttpSettings_ProxyPort_FieldLabel: 'HTTP proxy port',
     System_HttpSettings_Authentication_Title: 'Authentication',
     System_HttpSettings_ExcludeHosts_FieldLabel: 'Hosts to exclude from HTTP/HTTPS proxy',
-    System_HttpSettings_ExcludeHosts_HelpText: 'Java http.nonProxyHosts glob-like expressions are supported (do not use \'|\' delimiter)',
+    System_HttpSettings_ExcludeHosts_HelpText: 'Java system property http.nonProxyHosts glob-like patterns are supported (one per line, no \'|\' hostname delimiters)',
```

Comment:
Not such adding "system property" makes it easier to understand, also the Oracle/Sun docs mention wildcards rather than globs. Perhaps change help text to: `'Accepts Java "http.nonProxyHosts" wildcard patterns (one per line, no \'|\' hostname delimiters)'` ?